exercises/primzahlen.py

```diff
@@ -0,0 +1,26 @@
+from time import *
+prim = [1, ]
```

Comment:
Definition einer Primzahl
> Eine natürliche Zahl, die genau zwei *voneinander verschiedene" Teiler hat: 1 und sich selbst.

Deshalb ist 1 keine Primzahl, weil es nur einen Teiler hat... ;)

```suggestion
prim = []
```